src/main/resources/config/application-prod.yml

```diff
@@ -53,8 +53,6 @@ spring:
             starttls.enable: false
     thymeleaf:
         cache: true
-    liquibase:
-        contexts: prod
```

Comment:
does this need to be removed? Would it not be overridden by the `e2e-prod-test` profile?

Reply:
I moved it to the apllication.yml, the reasoning being that it is the default unless overriddedn by one of the dev profiles. Don't feel super strongly on this.